canonical-web-and-design/vanilla-framework
# Apply shallow strip padding to large lists

## Done

Fixes  #1917

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ or [demo]
- Verify lists have padding top and bottom matching shallow strip

![image](https://user-images.githubusercontent.com/2741678/84059486-9e1a0680-a9b2-11ea-92a4-82e465fdd25c.png)
